main/gconf/APKBUILD

```diff
@@ -22,7 +20,10 @@ build() {
 		--build=$CBUILD \
 		--host=$CHOST \
 		--prefix=/usr \
-		--sysconfdir=/etc
+		--sysconfdir=/etc \
+		--disable-static \
```

Comment:
Splitting out `-static` would be better.

Reply:
I don't think there is much value in splitting out static libraries of gnome stuff